File: bin/local.js
const BBK = require('../');
const options = require('../lib/option');

// const options = {
//     localAddress: '127.0.0.1',
//     localPort: 1080,
//     serverAddress: '127.0.0.1',
//     serverPort: 5900,
//     websocketUri: '/websocket',
//     password: 'p@ssword',
//     method: 'aes-256-cfb',
// };

var client = new BBK.Client(options);

client.bootstrap();

File: local.go
package main

import bbk "bbk/src"

func main() {
	client := bbk.Client{
		ListenAddr:   "127.0.0.1",
		ListenPort:   1090,
		FillByte:    0,
		LogLevel:     "info",
		Method:       "aes-256-cfb",
		Password:     "p@ssword",
		WebsocketUrl: "ws://127.0.0.1:5900/websocket",
		Ping:         true,
	}
	client.Bootstrap()
}
